engineering-toolkit/eslint-config/eslint.config.js

```diff
@@ -0,0 +1,5 @@
+import { concat } from "eslint-flat-config-utils";
+
+import { ecma, style } from "./src/index.js";
+
+export default concat(ecma(), style());
```

Comment:
Offtop: does this concat does any more advanced rule concatenation, or only resolves promises and pushes them back into an array?

Reply:
Ok, good to know 😄

Reply:
There is a `composer` or something like that, it's more sophisticated